ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -13,10 +13,26 @@
 # limitations under the License.
 
 
-EXIT:
-  description: "Illuminated exit signs"
-  is_abstract: true
+LT_SS:
+  description: "Standard light."
+  is_canonical: true
   implements:
   - LT
+  - SS
 
+LT_SS_DC:
+  description: "Standard light with dimming control."
+  is_canonical: true
+  implements:
+  - LT
+  - SS
+  - DC 
 
+LT_SS_DC_SC:
+  description: "Standard light with dimming control."
```

Comment:
and...

Reply:
...and its birthday is friday? Theres no special difference between this and the other types, just different fields. How would you like the description adjusted with that in mind?

Reply:
It just seemed to have a lot of sub-types, thought maybe there was some more in there.

Reply:
conclusion: remove SS from name and implements